src/components/Navigation/Navigation.scss

```diff
@@ -406,16 +347,13 @@ $secondary-item-font-size: rem(15px);
 
   .Action {
     @include unstyled-button;
-    @include recolor-icon(
-      var(--p-icon, color('ink', 'lightest')),
-      var(--p-override-transparent, color('white'))
-    );
+    @include recolor-icon(var(--p-icon), var(--p-override-transparent));
```

Comment:
I think you can remove `var(--p-override-transparent)` from this line. It was only necessary for when we had multicolored icons